src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/DefaultRazorFormattingServiceTest.cs

```diff
@@ -543,5 +543,44 @@ public async Task OnTypeCloseAngle_OutsideRazorBlock_DoesNotCloseTextTag()
 ",
 character: ">");
         }
+
+        [Fact]
+        public async Task OnTypeStar_ClosesRazorComment()
+        {
+            await RunFormatOnTypeTestAsync(
+input: @"
+@|
+",
+expected: $@"
+@* {LanguageServerConstants.CursorPlaceholderString} *@
+",
+character: "*");
+        }
+
```

Comment:
```suggestion

        [Fact]
        public async Task OnTypeStar_ClosesRazorCommentWithAdditionalText()
        {
            await RunFormatOnTypeTestAsync(
input: @"
@| foobar
",
expected: $@"
@* {LanguageServerConstants.CursorPlaceholderString} *@ foobar
",
character: "*");
        }

```

Reply:
Sure will add. Can't commit here directly because `foobar` is a policheck violation 😄